tests/common/devices/sonic.py

```diff
@@ -1493,6 +1493,20 @@ def get_ip_route_summary(self, skip_kernel_tunnel=False):
                 ipv4_summary['Totals']['routes'] -= ipv4_route_kernel_count
                 ipv4_summary['Totals']['FIB'] -= ipv4_route_kernel_count
 
+        if skip_kernel_linkdown is True:
```

Comment:
Is this caused by a change in FRR’s behavior, or is it an FRR issue?
If it's an issue, could we create an FRR issue to track it? thanks.